.byebug_history

```diff
@@ -0,0 +1,9 @@
+exit
```

Comment:
move to gitignore

Reply:
@VKuzmich not fixed :disappointed:

Reply:
fixed. also i removed tmp folder that was used for rubycritic